File: JavaStudio/app/src/main/java/com/example/javastudio/MainActivity.java
package com.example.javastudio;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    int clickCount = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //메시지를 잠시 보여준다.
        Toast.makeText(getApplicationContext(), "프로그래밍을 시작해보자!", Toast.LENGTH_LONG).show();

        // 레이아웃에 선언된 뷰에 클릭 이벤트 리스너를 등록한다.
        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v){
                clickCount = clickCount + 1;

                Toast.makeText(getApplicationContext(), "프로그래밍을 시작해보자! " + clickCount, Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: lotto/app/src/main/java/com/example/lotto/MainActivity.kt
package com.example.lotto

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import java.util.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Toast.makeText(applicationContext, "Main 액티비티 입니다.", Toast.LENGTH_SHORT).show()
    }

    fun getRandomLottoNumber(): Int{
        return Random().nextInt(45) + 1
    }

    fun gRLNs(): MutableList<Int>{
        // 무작위로 생성된 로또 번호를 저장할 가변 리스트 생성
        val lottoNumbers = mutableListOf<Int>()

        // 6번 반복하는 for문
        for(i in 1..6){
            lottoNumbers.add(getRandomLottoNumber())
        }

        return lottoNumbers
    }

}